### ISSUE TITLE
[convenience] Subscribe Tiles entered

### ISSUE THREAD

**Issue by zewa666** [Annoraaq/grid-engine#244]
Hey there @Annoraaq.

This one falls under the category of convenience methods, as such fully understandable if you dont see this as a good fit or bloat of public API.

 It would be great if one could setup a subscription if one or more positions are being stepped on by one or more characters in order to trigger an action.

E.g I have a randomly generated dungeon where if the player steps on the stairs he exits the dungeon and gets into another scene.

I'm currently doing:

```ts
this.gridEngine.positionChangeFinished()
      .pipe(
        takeUntil(this.destroy$),
        filter(t => t.charId === PLAYER_KEY
          && t.enterTile.x === myTile.x
          && t.enterTile.y === myTile.Y))
      .subscribe(() => {
     ...
```

works great but could perhaps be streamlined to something like `stepOn(chars: string[], tiles: Position[]): Obs<{char: string, tile: Position}>`.

**Comment by Annoraaq** [comment]
👍 seems like a reasonable helper.

**Comment by Annoraaq** [comment]
closed with 2.14.0
